src/panel-container-element.js

```diff
@@ -54,16 +54,21 @@ class PanelContainerElement extends HTMLElement {
         if (visible) { this.hideAllPanelsExcept(panel) }
       }))
 
-      if (panel.autoFocus) {
-        const modalFocusTrap = focusTrap(panelElement, {
+      if (panel.autoFocus !== false) {
+        const focusOptions = {
           // focus-trap will attempt to give focus to the first tabbable element
           // on activation. If there aren't any tabbable elements,
           // give focus to the panel element itself
           fallbackFocus: panelElement,
           // closing is handled by core Atom commands and this already deactivates
           // on visibility changes
           escapeDeactivates: false
-        })
+        }
+
+        if (panel.autoFocus !== true) {
```

Comment:
What if someone deliberately sends `undefined` or `null`?  It seems like those values will get through these two `panel.autoFocus` guards and possibly cause an error in `focusTrap`.

Reply:
Nice catch! I updated it so any falsy value will be treated the same as false